db/user.go

```diff
@@ -221,3 +222,45 @@ func SyncUsers(course *model.Course, newUsers map[string]*model.User,
 
     return syncResult, nil;
 }
+
+// Insert method that takes a course, email [] string and resolves based on *, roles, emails to all emails
```

Comment:
Accidental comment?

This looks like a note to yourself.
This function could use a nice comment that describes what it does.